modules/webgl/src/classes/program-configuration.js

```diff
@@ -96,6 +97,7 @@ export default class ProgramConfiguration {
 
     const attributeInfo = {location, name, accessor: new Accessor(accessor)}; // Base values
     this.attributeInfos.push(attributeInfo);
+    this.attributeInfosByLocation[location] = attributeInfo; // For quick location based lookup
```

Comment:
Comment on the difference between `attributeInfos` and `attributeInfosByLocation` and how they differ in this edge case?